docs/2-setting_up_the_entities.md

```diff
@@ -65,5 +65,33 @@ doctrine:
             Symfony\Component\Security\Core\User\UserInterface: App\Entity\User
 ```
 
+### C - Extend the repository
+
+You wil need to extend the repository on your project. You can see an exemple of 'TicketRepository' below. 
```

Comment:
maybe : You can see an example with \`\`TicketRepository\`\` below.

or

You can see the example of \`\`TicketRepository\`\` below.